components/collector/src/source_collectors/grafana_k6/performancetest_duration.py

```diff
@@ -0,0 +1,24 @@
+"""Grafana k6 performancetest duration collector."""
+
+from typing import cast
+
+from base_collectors import JSONFileSourceCollector
+from collector_utilities.exceptions import CollectorError
+from collector_utilities.type import JSONDict, Value
+from model import Entities, SourceResponses
+
+
+class GrafanaK6PerformanceTestDuration(JSONFileSourceCollector):
+    """Collector for the performance test duration."""
+
+    async def _parse_value(self, responses: SourceResponses, included_entities: Entities) -> Value:
+        """Override to parse the performance test durations from the responses and return the sum in minutes."""
+        duration = 0.0
+        for response in responses:
+            json = await response.json(content_type=None)
+            try:
+                duration += cast(JSONDict, json)["state"]["testRunDurationMs"]
+            except KeyError as error:
+                message = 'Could not find an object {"state": {"testRunDurationMs": value}} in summary.json.'
```

Comment:
Is this informative enough for the end-user?

Reply:
I hope so. The Grafana k6 docs are pretty sparse on the summary.json format so I'm not sure if the `state` is going to be missing often.